### PR TITLE
introduce arcgis press

### PR DESCRIPTION
**Pull request by steveoh**
i can send you the secrets when you are ready

### CODE REVIEW

File: GruntFile.js [hunk 1]
```diff
@@ -60,19 +69,7 @@ module.exports = function (grunt) {
...
-                username: '',
-                password: ''
-            },
-            prod: {
-                host: '',
-                username: '',
-                password: ''
-            }
-        };
+        grunt.log.ok();
```
Comment (stdavis) [inline GruntFile.js]: What's this for?

**Comment by stdavis** [comment]
Please send me the secrets at your convenience.

File: GruntFile.js (hunk 1 above)
Comment (steveoh) [inline GruntFile.js]: mainly to not have an empty block

**Comment by stdavis** [comment]
Thanks for doing this, BTW.

**Pull request merged**